package.json

```diff
@@ -33,77 +33,68 @@
   },
   "scripts": {
     "build": "unbuild",
-    "dev": "jiti scripts/watch.ts --cache",
+    "dev": "nr dev:vite",
+    "release": "release-it",
     "lint": "eslint --ext .js,.vue,.ts,.tsx .",
     "lint:fix": "eslint --ext .js,.vue,.ts,.tsx . --fix",
-    "test:types": "tsc --build tsconfig.json",
     "test:unit": "vitest run",
     "test:coverage": "vitest run --coverage",
-    "test": "pnpm test:types && pnpm test:unit && pnpm test:coverage",
-    "dev:docs": "vitepress dev docs",
-    "build:docs": "vitepress build docs",
-    "serve:docs": "vitepress serve docs",
-    "publish:ci": "jiti scripts/publish.ts",
-    "release": "jiti scripts/release.ts && git push --follow-tags",
-    "dev:demo": "vite",
-    "build:demo": "vite build",
-    "serve:demo": "vite serve demo/dist",
-    "prepare:nuxt": "nuxt-module-build --stub && nuxi prepare playground",
-    "build:nuxt": "nuxt-module-build",
-    "dev:nuxt": "nuxi dev nuxt-playground",
-    "build:nuxt:playground": "nuxi build playground"
+    "test": "pnpm test:unit && pnpm test:coverage",
+    "prepare": "nr prepare:nuxt",
+    "__": "__",
+    "dev:nuxt": "(cd playgrounds/nuxt && nr dev:nuxt)",
+    "build:nuxt": "(cd playgrounds/nuxt && nr build:nuxt)",
+    "generate:nuxt": "(cd playgrounds/nuxt && nr preview:nuxt)",
+    "dev:ssg": "(cd playgrounds/vite-ssg && nr dev:ssg)",
+    "build:ssg": "(cd playgrounds/vite-ssg && nr build:ssg)",
+    "preview:ssg": "(cd playgrounds/vite-ssg && nr preview:ssg)",
+    "dev:vite": "(cd playgrounds/vite && nr dev:vite)",
+    "build:vite": "(cd playgrounds/vite && nr build:vite)",
+    "preview:vite": "(cd playgrounds/vite && nr preview:vite)",
+    "dev:docs": "(cd docs && nr dev:docs)",
+    "build:docs": "(cd docs && nr build:docs)",
+    "preview:docs": "(cd docs && nr preview:docs)",
+    "prepare:nuxt": "(cd playgrounds/nuxt && nr prepare:nuxt)"
   },
   "files": [
     "dist/**/*",
     "LICENSE",
     "README.md"
   ],
   "dependencies": {
-    "@vueuse/core": "^8.2.3",
-    "@vueuse/shared": "^8.2.3",
-    "csstype": "^3.0.11",
-    "framesync": "^6.1.0",
-    "popmotion": "^11.0.3",
-    "style-value-types": "^5.1.0",
-    "vue-demi": "*"
+    "@vueuse/core": "^9.2.0",
+    "@vueuse/shared": "^9.2.0",
+    "csstype": "^3.1.0",
+    "framesync": "^6.1.2",
+    "popmotion": "^11.0.5",
+    "style-value-types": "^5.1.2"
   },
   "peerDependencies": {
-    "@nuxt/kit": "npm:@nuxt/kit-edge@latest",
-    "@vue/composition-api": "^1.4.1",
-    "vue": "^2.0.0 || >=3.0.0-rc.0"
-  },
-  "peerDependenciesMeta": {
```

Comment:
shouldn't this be kept and also make @nuxt/kit optional? It currently warns on a vue project

Reply:
Seems fixe! 👍